css/main.scss

```diff
@@ -7,7 +7,7 @@
 
 // Our variables
 $base-font-family: Helvetica, Arial, sans-serif;
-$base-font-size:   16px;
+$base-font-size:   16;
```

Comment:
Saw a font size difference - might this be the guilty change? If the `px` needs to go, maybe try with 18 as the base font size.